azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -585,8 +587,10 @@ def _clean_up_folders(
                 f"monikers: [{monikers}]"
             assert not level_1["files"]
 
-    if acstor_namespace:
-        level_1 = walk_result.pop(path.join(BASE_ZIP_PATH, acstor_namespace))
+    if acstor_namespace or acs_namespace:
```

Comment:
nit: variable for "acstor_namespace or acs_namespace" since it's been used twice?

Reply:
eh meh... would just add more variables/confusion